care/utils/queryset/filters.py

```
@@ -0,0 +1,42 @@
+
+    Examples:
+        # Sort patients by priority status: ['critical', 'urgent', 'stable']
+        patients = Patient.objects.annotate(
+            priority_order=sort_index('status', ['critical', 'urgent', 'stable'])
+        ).order_by('priority_order')
+    """
+    if not order:
+        return models.Value(0, output_field=models.IntegerField())
+
+    if not isinstance(order, list):
+        raise TypeError("order must be a list")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)